src/modelgenerator.cpp

```diff
@@ -81,6 +79,7 @@ void model_generator::load_model(const std::string &model_file)
 
     for (auto child : root)
     {
+// Missing elements: "ReferenceType" "DataType" "Method" "Variable" "Property" "Dictionary" "View"
```

Comment:
I like the TODO markers so it is easier to find them again later and also because many IDEs support listing them
```suggestion
        // TODO: Missing elements: "ReferenceType" "DataType" "Method" "Variable" "Property" "Dictionary" "View"
```